package.json

```diff
@@ -26,7 +26,7 @@
     "sanitize-filename": "^1.6.3",
     "traverse": "^0.6.6",
     "typedarray-to-buffer": "^4.0.0",
-    "typedfastbitset": "~0.1.5"
+    "typedfastbitset": "~0.2.1"
```

Comment:
we should probably do ^ here again, will fix that after merge.

Reply:
I actually prefer the tilde by default, since it's common to get breaking changes in the minor num, in the npm ecosystem. That was the case of typedfastbitset 0.2.0 too.

Reply:
Oh, will revert then. Just pushed a minor change